packages/backend/src/types.ts

```diff
@@ -21,14 +22,17 @@ export interface ProjectAdapter {
 }
 
 export interface DbtClient {
-    installDeps(): Promise<void>;
-    getDbtManifest(): Promise<{
-        version: DbtManifestVersion;
-        results: DbtRpcGetManifestResults;
-    }>;
-    getDbtCatalog(): Promise<DbtRpcDocsGenerateResults>;
-    getDbtPackages?(): Promise<DbtPackages | undefined>;
-    test(): Promise<void>;
+    installDeps(dbtVersion: SupportedDbtVersions): Promise<void>;
+    getDbtManifest(
+        dbtVersion: SupportedDbtVersions,
+    ): Promise<DbtRpcGetManifestResults>;
+    getDbtCatalog(
+        dbtVersion: SupportedDbtVersions,
+    ): Promise<DbtRpcDocsGenerateResults>;
+    getDbtPackages?(
+        dbtVersion: SupportedDbtVersions,
+    ): Promise<DbtPackages | undefined>;
+    test(dbtVersion: SupportedDbtVersions): Promise<void>;
```

Comment:
Since this is a class, perhaps the `dbtVersion` should be passed in the constructor. That way we only need to pass it once and is guarantied that all methods of the same instance use the same version.